### PR TITLE
Project Cleanup and Navbar Setup

### PR DESCRIPTION
**Pull request by dtleight**
These changes provide a base to work off of for the home page and set up a navbar with no page navigation attached.
image

### CODE REVIEW

File: lib/Widgets/GridText.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+
+class GridText extends StatelessWidget
+{
+  final String number;
+  final String text;
+  GridText(this.number, this.text);
+
```
Comment (MattLaraia) [inline lib/Widgets/GridText.dart:8]: maybe add @override

Comment (MattLaraia) [inline lib/Widgets/GridText.dart:8]: Maybe add @override

File: lib/Widgets/ImageHelper.dart [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+class ImageHelper extends StatelessWidget
+{
+  final String url;
+  final String text;
+  ImageHelper(this.url, this.text);
+
```
Comment (MattLaraia) [inline lib/Widgets/ImageHelper.dart:8]: @override

**Review by MattLaraia — COMMENTED** [review]
I don't know if forgetting @override is really worth approving or requesting changes so I went with the comment option. Please tell me at some point if I am supposed to choose 1 of the other options

File: lib/Pages/UserPage.dart [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import '../Widgets/GridText.dart';
+import '../Widgets/ImageHelper.dart';
+
+class UserPage extends StatelessWidget {
+  UserPage();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.lightBlue,
```
Comment (USabah) [inline lib/Pages/UserPage.dart:13]: Should we set the background and nav color to theme variables?

**Review by USabah — APPROVED** [review]

File: lib/Pages/UserPage.dart (hunk 3 above)
Comment (dtleight) [reply lib/Pages/UserPage.dart:13]: Yes, this will be handled in a later PR.

**Review by JordanMiner — APPROVED** [review]

**Pull request merged**